Gemfile

```diff
@@ -12,21 +12,26 @@ gem "dynamoid"
 gem "fog-aws"
 gem "govuk_app_config", "~> 2.1.1"
 gem "govuk_publishing_components", "~> 21.42.0"
+gem "json"
```

Comment:
You could remove this line I think, since `json` is a [default Ruby gem](https://stdgems.org/json/).

Reply:
Hi @bilbof - just adding a bit of context on this.  You're absolutely, right `json` is a default gem and shouldn't need to be added here.  However, without this in the Gemfile - I found I was getting some odd warning messages when running the tests.  These go away with it's inclusion :-/

Reply:
> These go away with it's inclusion

You might need to `require 'json'` in the files where you use `json`, though I'm not sure what warnings you were seeing.